src/loaders/workspace_downloader/workspace_downloader.py

```diff
@@ -0,0 +1,356 @@
+"""
+usage: workspace_downloader.py [-h] --workspace_id WORKSPACE_ID [--root_dir ROOT_DIR] [--kb_base_url KB_BASE_URL] [--workers {1,2,3,4}]
+                               [--token_filename TOKEN_FILENAME] [--delete_job_dir]
+
+PROTOTYPE - Download genome files from the workspace service (WSS).
+
+options:
+  -h, --help            show this help message and exit
+
+required named arguments:
+  --workspace_id WORKSPACE_ID
+                        Workspace addressed by the permanent ID
+
+optional arguments:
+  --root_dir ROOT_DIR   Root directory.
```

Comment:
Do the defaults not show up in the help message? I thought they did

Reply:
ah, they are not showing up when I do -h

Reply:
Fair enough. In that case can you add the defaults to the help text strings?

Reply:
This looks like an easy way to do it: https://stackoverflow.com/a/67208041/643675

Reply:
Updated!

Reply:
:+1:

Reply:
:+1: